staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -255,6 +255,51 @@ type PodFailurePolicy struct {
 	Rules []PodFailurePolicyRule `json:"rules" protobuf:"bytes,1,opt,name=rules"`
 }
 
+// SuccessPolicy describes when a Job can be declared as succeeded based on the success of some indexes.
+type SuccessPolicy struct {
+	// Rules represents the list of alternative rules for the declaring the Jobs
```

Comment:
```suggestion
	// rules represents the list of alternative rules for the declaring the Jobs
```

We use the json names, because they are used in the autogenerated reference docs.

Reply:
It makes sense.

Reply:
Done.